### DISCUSSION TITLE
How to get files from local NuGet source into heat in Wix4 projects?

### DISCUSSION THREAD

**Discussion by nikryden** [wixtoolset/issues#7269] [Questions] [answered]
Before in Wix 3 I get files from our local nuget package source and extract them in a Extra folder so It's could be added to the heat. To do this I change the path in  nuget.config.  This can not be done any more. 
NuGet do not pack up nuget files in the selected package folder after I convert to WIX 4 project with HeatWave  for VS2022 version 0.9.4. Nothing else is change in the projects.

I have tried to start a new wix 4 project and get the same result. Do I have to do this in some other way in WIX 4.  Have tried to find any thing about this on the web with no luck.

I don't know if this has any thing to do with this but with WIX3 project i could do Update-Package –reinstall
but with WIX 4 i get this error when run the same 
```
PM> Update-Package –reinstall
The `-Reinstall` parameter does not apply to PackageReference based projects `TestWix4MSI`
```

**Comment by robmen** [comment, marked as answer]
As the error message tells you, WiX v4 adopts the PackageReference mechanism for referencing NuGet packages. If that is new to you, you'll want to read up on it. The [GeneratePathProperty](https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#generatepathproperty) is probably the most relevant.

**Reply by nikryden** [reply]
Thanks that was a big help.
